README.md

```diff
@@ -128,6 +128,7 @@ The writes are processed in batches which are configurable by `WriteOptions`:
 | max_retries | the number of max retries when write fails | 5 |
 | max_retry_delay | maximum delay when retrying write in milliseconds | 180000 |
 | exponential_base | the base for the exponential retry delay, the next delay is computed as `retry_interval * exponential_base^(attempts - 1) + random(jitter_interval)` | 5 |
+| batch_abort_on_exception | the batching worker will be aborted after failed retry strategy | false |
```

Comment:
Should probably default to `true` since that is what it has been if people are relying on that.

Reply:
The `Thread.abort_on_exception = true` was introduced accidentally in release `1.7.0` by #47. The default strategy should be **best-effort** as you mentioned.

- https://github.com/influxdata/influxdb-client-ruby/blob/281ce2b06c7cd26b607b8b0e67e7f203afedab58/lib/influxdb2/client/worker.rb#L34

Reply:
In that case, i've approved the changes! Thank you so much for such a fast response!

Reply:
You are welcome.